src/DssSpell.sol

```diff
@@ -106,226 +99,292 @@ contract DssSpellAction is DssAction {
-            input[0] = PoolConfiguratorLike.InitReserveInput({
-                aTokenImpl: SPARK_ATOKEN_IMPL,
-                stableDebtTokenImpl: SPARK_STABLE_DEBT_TOKEN_IMPL,
-                variableDebtTokenImpl: SPARK_VARIABLE_DEBT_TOKEN_IMPL,
-                underlyingAssetDecimals: GemAbstract(token).decimals(),
-                interestRateStrategyAddress: SPARK_INTEREST_RATE_STRATEGY,      // Dummy strategy - compare to other borrow-disabled asset like sDAI
-                underlyingAsset: token,
-                treasury: SPARK_TREASURY,
-                incentivesController: address(0),
-                aTokenName: "Spark GNO",
-                aTokenSymbol: "spGNO",
-                variableDebtTokenName: "Spark Variable Debt GNO",
-                variableDebtTokenSymbol: "variableDebtGNO",
-                stableDebtTokenName: "Spark Stable Debt GNO",
-                stableDebtTokenSymbol: "stableDebtGNO",
-                params: ""
-            });
-            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).initReserves(input);
-            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).configureReserveAsCollateral({
-                asset: token, 
-                ltv: 20_00,
-                liquidationThreshold: 25_00,
-                liquidationBonus: 110_00
-            });
-            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).setDebtCeiling(token, 5 * MILLION * HUNDRED);
-
-            address[] memory tokens = new address[](1);
-            tokens[0] = token;
-            address[] memory oracles = new address[](1);
-            oracles[0] = SPARK_GNO_ORACLE;
-            AaveOracleLike(SPARK_AAVE_ORACLE).setAssetSources(
-                tokens,
-                oracles
-            );
-        }
-
-        // Reduce Maker Protocol GNO Debt Ceiling to Zero
-        (,,,uint256 line,) = VatLike(MCD_VAT).ilks("GNO-A");
-        DssExecLib.removeIlkFromAutoLine("GNO-A");
-        DssExecLib.decreaseIlkDebtCeiling("GNO-A", line / RAD, true);
-
-        // Bump the chainlog
-        DssExecLib.setChangelogVersion("1.14.12");
+        // --- BlockTower Vault Debt Ceiling Adjustments ---
+        // Poll: https://vote.makerdao.com/polling/QmPMrvfV#poll-detail
+        // Forum: https://forum.makerdao.com/t/blocktower-credit-rwa-vaults-parameters-shift/20707
+
+        // Decrease the Debt Ceiling (line) of BlockTower S1 (RWA010-A) from 20 million Dai to zero Dai.
+        DssExecLib.setIlkDebtCeiling("RWA010-A", 0);
+        // Decrease the Debt Ceiling (line) of BlockTower S2 (RWA011-A) from 30 million Dai to zero Dai.
+        DssExecLib.setIlkDebtCeiling("RWA011-A", 0);
+        // Increase the Debt Ceiling (line) of BlockTower S3 (RWA012-A) from 30 million Dai to 80 million Dai.
+        // Note: Do not increase global Line because there is no net change from these operations
+        DssExecLib.increaseIlkDebtCeiling("RWA012-A", 50 * MILLION, /* do not increase global line */ false);
```

Comment:
The comment before the param name is redundant with the line above.
```suggestion
        DssExecLib.increaseIlkDebtCeiling("RWA012-A", 50 * MILLION, /* global = */ false);
```

Reply:
Also that `50*MILLION` should probably be extracted into a constant like `RWA012_A_LINE`.

Reply:
^ Agree with above, and we could use setIlkDC rather than increase if we aren't touching global Line

Reply:
Hmmm, I guess you have a point.

Checking the archives, I found the following (absence of) standard:

![image](https://github.com/makerdao/spells-goerli/assets/112016538/b1717c76-50d7-4914-9d98-9040261fe10a)

Apparently for RWAs we usually extract values into constants, but for crypto-native assets, we used numbers directly.

When doing it for `AutoLine`, the standard is to use numbers directly:

![image](https://github.com/makerdao/spells-goerli/assets/112016538/ae922e61-6597-49af-b697-47c4162ed3b5)